packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -104,21 +103,21 @@ _ref:
 
             ## Menus
 
-            Menu objects are objects with links to pages. They are filtered to only show pages that the user is authorized to see and are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+            Menu objects describe links to pages, within the app or external. Menu lists are filtered to only show pages that the user is authorized to see as a result of public, private or [role based access controlled (RBAC)](/roles) pages. Blocks such as `PageSiderMenu` render menu links and by default if no menu defined, a default menu is created, containing links to all pages defined in the app.
```

Comment:
This is nice.

I would split the default menu sentance to a new sentance:
`render menu links. If no menu is defined, a default menu is created, containing links to all pages defined in the app.`

Reply:
This definitely reads better.
Change made in 386463869c78c494c05636e1e435419c96917b02